src/common/models/Projection.ts

```diff
@@ -550,7 +550,7 @@ export class Projection {
           row &&
           this.totalICUCapacity &&
           this.totalICUCapacity > 0 &&
-          row.ICUBedsInUse > 0
+          row.ICUBedsInUse != null
```

Comment:
this will catch undefined as well (here and below).

Reply:
Eh. It was sorta' intentional because I just don't trust our code not to accidentally have `undefined`s in it right now (since it's half JavaScript, etc.) but it _seems_ like `!== null` is sufficient.  Changed.